### PR TITLE
Update STORAGE.md

### PR DESCRIPTION
**Pull request by AbdulShan**
Apart from advertisement, both are free and useful for academics/jobs.

### CODE REVIEW

File: STORAGE.md [hunk 1]
```diff
@@ -183,6 +183,8 @@ http://104.131.175.196:8080/, http://70.88.180.169:8088/, http://207.237.122.161
 * ⭐ **[GeeksforGeeks](https://www.geeksforgeeks.org/)**
 * ⭐ **[W3Schools](https://www.w3schools.com/)** - [Offline Version](https://github.com/Ja7ad/W3Schools)
 * ⭐ **[Build your own X](https://github.com/codecrafters-io/build-your-own-x)**
+* ⭐ **[TakeuForward](https://takeuforward.org)**  DSA / SDE / Roadmap
```
Comment (taskylizard) [inline STORAGE.md:186]: Use `-` to delimit these, see the above items for example.

**Review by taskylizard — CHANGES_REQUESTED** [review]

**Comment by AbdulShan** [comment]
used - to delimit

File: STORAGE.md (hunk 1 above)
Comment (AbdulShan) [reply STORAGE.md]: > Use `-` to delimit these, see the above items for example.

used - to delimit

**Review by SnowzNZ — COMMENTED** [review]
Worth starring?

**Comment by nbats** [comment]
we'll take a look thank you

**Pull request closed without merging**